ods_ci/tests/Resources/Page/Pipelines/OpenShiftPipelines.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
```

Comment:
probably this file can go under `ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.robot`

Reply:
This is related to the OpenShift Pipelines Operator, not Data Science Pipelines in RHODS. 

In my opinion, it could be in a new resource file to consolidate keywords to manage that operator: ods_ci/tests/Resources/Operators/OpenShiftPipelines

```
Install Red Hat OpenShift Pipelines Operator  version=1.9.0
Wait Until OpenShift Pipelines Operator Is Deployed
Is OpenShift Operator Installed
Get OpenShift Operator Version
```
Beware there are already methods to implement this using the UI in in ods_ci/tests/Resources/Page/OCPDashboard/OperatorHub/OperatorHub.robot and ods_ci/tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot


For keywords related to RHODS Data Science Pipelines, I agree with @bdattoma that we could concentrate them in ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.robot

Reply:
The files split is done